input/pagecontent/conformance-profiles.md

```diff
@@ -82,42 +82,45 @@ An mCODE data element is **required** if any of the following criteria are met:
+| [Choice \[x\] type](https://www.hl7.org/fhir/stu3/formats.html#choice), MS flag on one or more choice | MUST support only the datatype choice(s) that are explicitly flagged | same |
+| [Reference data type](https://www.hl7.org/fhir/references.html#2.3.0), no MS flag on any referenced resource or profile | MUST support all resources or profiles in the reference that are in Sender's capability statement | MUST support all resources or profiles in the reference unless outside the scope of the Receiver's capability statement |
+| [Reference data type](https://www.hl7.org/fhir/references.html#2.3.0), MS flag on one or more referenced types | MUST support only the resources or profiles in the reference that are explicitly flagged, and only if they are in the Sender's capability statement | MUST support only the resources or profiles in the reference that are explicitly flagged, and only if they are in the Receiver's capability statement |
+| [Backbone data type](https://www.hl7.org/fhir/backboneelement.html#2.29.0) | No support expectation on sub-elements unless they are explicitly flagged  | same |
+| [Sliced array](https://www.hl7.org/fhir/profiling.html#slicing), no MS flag on any slice | MUST support populating the array, and SHOULD populate every slice for which the server might possess data | MUST support any potential array contents, unless outside the scope of the Receiver's capability statement |
+| [Sliced array](https://www.hl7.org/fhir/profiling.html#slicing) type, MS flags on one or more slices | Support only the slices that are explicitly flagged, and only if they are in the scope of the Sender's capability statement | same |
+{: .grid }
 
-##### Viewing Must Support Flags
-To see which elements have <span style="padding-left: 3px; padding-right: 3px; color: white; background-color: red" >S</span> flags, consult the "Snapshot Table" view of the profile. The "Differential Table" view hides <span style="padding-left: 3px; padding-right: 3px; color: white; background-color: red" >S</span> flags inherited from the parent profile. The "Snapshot Table (Must Support)" view reflects the IG Publisher's interpretation of how <span style="padding-left: 3px; padding-right: 3px; color: white; background-color: red" >S</span> flags translate to support requirements, which may or may not coincide with the US Core/mCODE interpretation.
+#### Non-Must Support Elements
 
-#### Interpretation of <span style="padding-left: 3px; padding-right: 3px; color: white; background-color: red" >S</span> Flags
+Data elements in mCODE that *do not have* MS obligations MAY be supported. If a non-obligatory element is supported, the implementation of that element MUST conform to the corresponding profile. Moreover, any data element that would reasonably be expected to conform to that mCODE data element SHOULD conform to that element.
```

Comment:
This is confusing. Needs examples to clarify.

Reply:
I added an example and simplified the language.